astropy/modeling/functional_models.py

```diff
@@ -91,6 +91,32 @@ class Gaussian1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
+    _bbox = 'auto'
+
+    @property
+    def bbox(self):
```

Comment:
Regardless, if there is a property like `bbox` could we call it `bounding_box`?  Although I think most of our users would get "bbox" pretty readily, most of this interface tries to err on the side of spelling things out if it isn't all that long (maybe one of the few exceptions is abbreviating "standard_deviation" as "stddev" :)